src/Symfony/Component/Workflow/Registry.php

```diff
@@ -23,19 +24,26 @@ class Registry
 
     /**
      * @param Workflow $workflow
-     * @param string   $className
+     * @param SupportStrategyInterface $supportStrategy
      */
-    public function add(Workflow $workflow, $className)
+    public function add(Workflow $workflow, SupportStrategyInterface $supportStrategy)
```

Comment:
So here is the mentioned BC break... Alternatively we could intrdoduce a second method or keep $className and introduce an additional *optional* parameter for support strategy? Would be ok with my use case and probably with most others as well?